.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] You can add this file to your gitingore file as it is simply a personal editor settings file. 👍